space_test.go

```diff
@@ -93,15 +93,43 @@ func (rest *TestSpaceREST) TestSuccessCreateSpace() {
 	assert.NotNil(t, created.Data.Links.Self)
 }
 
+func (rest *TestSpaceREST) TestSuccessCreateSpaceWithDescription() {
```

Comment:
It's already there in `TestSuccessCreateSpace`, which is the default case. This is the new test.